pkg/services/apiserver/options/grafana-aggregator.go

```diff
@@ -88,7 +89,5 @@ func (o *GrafanaAggregatorOptions) ApplyTo(aggregatorConfig *aggregatorapiserver
 	}
 	genericConfig.MergedResourceConfig = mergedResourceConfig
 
-	genericConfig.PostStartHooks = map[string]genericapiserver.PostStartHookConfigEntry{}
```

Comment:
why was this change needed?

Reply:
this is already being done in the beginning of this method, it was just repeated [twice](https://github.com/grafana/grafana/blob/main/pkg/services/apiserver/options/grafana-aggregator.go#L49). So its not easy to see in the diff.